internal/reader/home_chain.go

```diff
@@ -115,24 +118,42 @@ func (r *homeChainPoller) poll() {
 }
 
 func (r *homeChainPoller) fetchAndSetConfigs(ctx context.Context) error {
-	var chainConfigInfos []ChainConfigInfo
-	err := r.homeChainReader.GetLatestValue(
-		ctx,
-		consts.ContractNameCCIPConfig,
-		consts.MethodNameGetAllChainConfigs,
-		primitives.Unconfirmed,
-		nil,
-		&chainConfigInfos,
-	)
-	if err != nil {
-		return err
+	var allChainConfigInfos []ChainConfigInfo
+	pageIndex := uint64(0)
+
+	for {
+		var chainConfigInfos []ChainConfigInfo
+		err := r.homeChainReader.GetLatestValue(
+			ctx,
+			consts.ContractNameCCIPConfig,
+			consts.MethodNameGetAllChainConfigs,
+			primitives.Unconfirmed,
+			map[string]interface{}{
+				"pageIndex": pageIndex,
+				"pageSize":  PageSize,
+			},
+			&chainConfigInfos,
+		)
+		if err != nil {
+			return err
```

Comment:
let's wrap that err `"get config index:1 pagesize:2: <err>"`

Reply:
u[pdated with error wrapping